packages/zip-it-and-ship-it/src/runtimes/node/parser/index.ts

```diff
@@ -120,7 +128,7 @@ const parseSource = (source: string) => {
 
 // Parses a JS/TS source and returns the resulting AST. If there is a parsing
 // error, it will get swallowed and `null` will be returned.
-export const safelyParseSource = (source: string) => {
+export const safelyParseSource = (source: string): Program | null => {
```

Comment:
This is an unrelated change I had to do to make the CI happy. All the jobs installing deps without lockfile were failing because of `tsc` errors:
- https://github.com/netlify/build/actions/runs/10510449486/job/29119105952?pr=5811#step:10:181